src/ui/common/src/components/integrations/dialogs/awsDialog.tsx

```diff
@@ -127,6 +237,15 @@ export const AWSDialog: React.FC<Props> = ({
       </Box>
       {value?.type === AWSCredentialType.AccessKey && accessKeyTab}
       {value?.type === AWSCredentialType.ConfigFilePath && configPathTab}
+      <Box sx={{ borderBottom: 1, borderColor: 'divider', mb: 2 }}>
+        <Tabs value={engineType} onChange={(_, value) => setEngineType(value)}>
```

Comment:
how do you send this field to backend? It appears we didn't call 'onUpdateField' when this option is changed?

Reply:
Do I need to send this to backend? This var is just to select which tab (we only have k8s for now) to show and it doesn't need to be sent to the server side.

Reply:
I see, I thought it's a part of the config, makes sense now